### ISSUE TITLE
Remove non-existent HP ID (2)

### ISSUE THREAD

**Issue by asmvernon** [monarch-initiative/mondo#1177]
**Mondo term (ID and Label):**
MONDO:0003924

**Xref that should be fixed (ID and label):**
Remove HP:0008196; it is not a valid HP ID

**Other comments:**
non-existent

**Comment by nicolevasilevsky** [comment]
HP:0008256 exists:
https://hpo.jax.org/app/browse/term/HP:0008256

Where were you looking?
